pkg/winc1500/patches/0003-Fix-unused-variables-warnings.-Fix-MODULE_GNRC_NETDE.patch

```diff
@@ -0,0 +1,285 @@
+From bcb8870369744e2fa6cf3a7ef1f793f57b2ec8cf Mon Sep 17 00:00:00 2001
+From: Bumsik Kim <k.bumsik@gmail.com>
+Date: Wed, 15 Aug 2018 14:29:34 -0400
+Subject: [PATCH 3/3] Fix unused variables warnings. Fix MODULE_GNRC_NETDEV
+ deprecation
+
+---
+ winc1500/bsp/include/nm_bsp.h                     |  2 +-
+ winc1500/bsp/source/nm_bsp_riot.c                 |  4 +++-
+ winc1500/bus_wrapper/source/nm_bus_wrapper_riot.c |  6 +++++-
+ winc1500/driver/source/m2m_hif.c                  |  9 ++++++++-
+ winc1500/driver/source/m2m_ota.c                  |  4 ++++
+ winc1500/driver/source/m2m_ssl.c                  |  2 ++
+ winc1500/driver/source/m2m_wifi.c                 |  8 ++++++++
+ winc1500/driver/source/nmbus.c                    |  2 ++
+ winc1500/driver/source/nmdrv.c                    |  2 ++
+ winc1500/driver/source/nmspi.c                    |  2 ++
+ winc1500/socket/source/socket.c                   | 14 ++++++++++++++
+ 11 files changed, 51 insertions(+), 4 deletions(-)
+
+diff --git a/winc1500/bsp/include/nm_bsp.h b/winc1500/bsp/include/nm_bsp.h
+index 5ffa8ad..9b89d8a 100644
+--- a/winc1500/bsp/include/nm_bsp.h
++++ b/winc1500/bsp/include/nm_bsp.h
+@@ -84,7 +84,7 @@
+  *                     Used as a data type of ISR function registered by \ref nm_bsp_register_isr
+  * @return         None
+  */
+-#if	defined(MODULE_GNRC_NETDEV)
++#if	defined(MODULE_NETDEV_ETH)
+ typedef void (*tpfNmBspIsr)(void *arg);
+ #else
+ typedef void (*tpfNmBspIsr)(void);
+diff --git a/winc1500/bsp/source/nm_bsp_riot.c b/winc1500/bsp/source/nm_bsp_riot.c
+index deced12..a20c367 100644
+--- a/winc1500/bsp/source/nm_bsp_riot.c
++++ b/winc1500/bsp/source/nm_bsp_riot.c
```

Comment:
Same as above, it seems that this file `nm_bsp_riot.c` was added for specific RIOT support. If it's added in your driver fork like it's done here there's no need for patching it (I mean it should be fixed upstream)